src/ol/typedefs.js

```diff
@@ -632,34 +605,31 @@ ol.interaction.SnapSegmentDataType;
  * A projection as {@link ol.proj.Projection}, SRS identifier string or
  * undefined.
  * @typedef {ol.proj.Projection|string|undefined} ol.proj.ProjectionLike
- * @api stable
  */
 ol.proj.ProjectionLike;
 
 
 /**
  * A function that takes an array of input data, performs some operation, and
  * returns an array of ouput data.  For `'pixel'` type operations, functions
- * will be called with an array of {@link ol.raster.Pixel} data and should
- * return an array of the same.  For `'image'` type operations, functions will
- * be called with an array of {@link ImageData
- * https://developer.mozilla.org/en-US/docs/Web/API/ImageData} and should return
- * an array of the same.  The operations are called with a second "data"
- * argument, which can be used for storage.  The data object is accessible
- * from raster events, where it can be initialized in "beforeoperations" and
- * accessed again in "afteroperations".
+ * will be called with an array of numbers representing pixel data (e.g.
+ * [R, G, B], [R, G, B, A] or [H, C, L]) and should return an array of the same.
```

Comment:
For `pixel` type operations, the function will be called with an array of pixels, where each pixel is an array of four numbers (`[r, g, b, a]`) in the range of 0 - 255.

If you don't want to use the word "pixel" in the docs, then you'd have to say the raster operation will be called with an array of arrays.  But it is not correct to say that it will be called with an array of numbers (and the length of the pixel arrays will always be 4 instead of sometimes 3 and sometimes 4).